pages/reference.mdx

```diff
@@ -309,6 +309,308 @@ The ability to view APM traces is only available to Discord employees.
 
 Discord's Gateway API is used for maintaining persistent, stateful WebSocket connections between your client and Discord servers. These connections are used for sending and receiving real-time events your client can use to track and update local state. The Gateway API uses secure WebSocket connections as specified in [RFC 6455](https://tools.ietf.org/html/rfc6455). For information on opening Gateway connections, please see the [Gateway API](/topics/gateway#connections) section.
 
+## Client Properties
+
+Client properties, or "super properties", contain tracking information about the client that is used for analytics and A/B testing purposes. These properties are sent in the Gateway [Identify](/topics/gateway-events#identify) payload and included with every HTTP request using the `X-Super-Properties` header as a base-64 encoded JSON object.
```

Comment:
"about the client that is used for analytics" might sound misleading
"every HTTP request" isn't clear enough that it's only outgoing requests imo
and I've never seen someone say "base-64" :p
```suggestion
Client properties, or "super properties", contain tracking information about the current client, used for analytics and A/B testing purposes. These properties are sent in the Gateway [Identify](/topics/gateway-events#identify) payload and included with every outgoing HTTP request using the `X-Super-Properties` header as a base64-encoded JSON object.
```

Reply:
Maybe also a brief note about anti-spam so people don't mistake it as an optional header for user accounts?